docs/introduction-to-ai.md

```diff
@@ -11,7 +11,19 @@ This chapter aims to introduce AI as the concept itself, its context along histo
 
 ## What is *Artificial Intelligence*?
 
-`TODO.`
+At this moment in time, Artificial Intelligence doesn't have an exact description, even when is a field with a relative long history. The main problem when defining Artificial Intelligence is that it implies that there is a Natural Ingelligence, so, when talking about humans we can think we have this chracteristic because we can manipulate our reality, but animals can do that too, in a different and more passive way, but they do.
+That's a question we have to keep wondering about, because there is no exact answer.
+
+One person that kept it simple, was Alan Turing, who proposed : The Turing Test. This would allow us to decide if a computer is intelligent. 
+The test consist on having a human talk to an entity on another room, the entity can be other human or a machine. If the person making the test can not differenciate if is having a conversation with a human when talking with the machine, then we can conclude the machine has artificial intelligence.
```

Comment:
- "The test ***consists in*** having a human ***talking**(...)"
- "If the person making the test can not ***differentiate*** if ***he/she*** is having (...)"
- "(...) we can conclude ***that*** the machine has artificial intelligence."